docs/website/root/mithril/mithril-network/architecture-c4.puml

```diff
@@ -0,0 +1,58 @@
+@startuml architecture
+title "Mithril network architecture"
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
+
+AddElementTag("external", $bgColor="#c1c1c1")
+
+Person(end_user, "End user", "Retrieves and verifies artifacts certified by the Mithril network")
+
+Container_Boundary(mithril_aggregator, "Mithril aggregator", $link="https://github.com/input-output-hk/mithril/tree/main/mithril-aggregator") {
+    Component(mithril_aggregator_runtime, "Aggregator Runtime", "FSM", "Maintains and transitions internal states of aggregator")
```

Comment:
Is 'Runtime' a proper name? If not, suggest using 'Aggregator runtime'.
In this sentence 'Maintains and transitions internal states of aggregator' suggest either:
'Maintains and transitions aggregator's internal states' or 'Maintains and transitions internal states of the aggregator'